studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx

```diff
@@ -155,49 +169,64 @@ const DiskSizeConfiguration = ({ disabled = false }: DiskSizeConfigurationProps)
         onCancel={() => setShowResetDbPass(false)}
         hideFooter
       >
         <Form
-          name={`disk-resize-form`}
+          name="disk-resize-form"
           initialValues={INITIAL_VALUES}
           validationSchema={diskSizeValidationSchema}
           onSubmit={confirmResetDbPass}
         >
           {({ isSubmitting }: { isSubmitting: boolean }) =>
             currentDiskSize >= maxDiskSize ? (
               <>
-                <Alert
-                  withIcon
-                  variant="warning"
-                  title={'Maximum manual disk size increase reached'}
-                >
+                <Alert withIcon variant="warning" title="Maximum manual disk size increase reached">
                   You cannot manually expand the disk size any more than {maxDiskSize} GB. If you
                   need more than this, contact us to learn more about the Enterprise plan.
                 </Alert>
               </>
             ) : (
               <>
                 <Modal.Content>
-                  <div className="w-full space-y-8 py-8">
+                  <div className="w-full space-y-4 py-6">
+                    <Alert
+                      withIcon
+                      variant={isAbleToResizeDatabase ? 'info' : 'warning'}
+                      title="This operation is only possible every 6 hours"
+                    >
+                      {isAbleToResizeDatabase
+                        ? `Upon updating your disk size, the next disk size update will only be available from ${dayjs().format(
+                            'DD MMM YYYY, HH:mm (ZZ)'
+                          )}`
+                        : `Your database was last resized at ${dayjs(lastDatabaseResizeAt).format(
+                            'DD MMM YYYY, HH:mm (ZZ)'
+                          )}. You can resize your database again in approximately ${formattedTimeTillNextAvailableResize}`}
+                    </Alert>
                     <InputNumber
+                      required
                       id="new-disk-size"
                       label="New disk size"
                       labelOptional="GB"
-                      required
+                      disabled={!isAbleToResizeDatabase}
                     />
                   </div>
                 </Modal.Content>
                 <Modal.Separator />
                 <Modal.Content>
-                  <div className="flex space-x-2 justify-between pb-2">
+                  <div className="flex space-x-2 justify-end pt-1 pb-3">
                     <Button type="default" onClick={() => setShowResetDbPass(false)}>
                       Cancel
                     </Button>
-                    <Button htmlType="submit" type="primary" loading={isSubmitting}>
+                    <Button
+                      htmlType="submit"
+                      type="primary"
+                      disabled={!isAbleToResizeDatabase}
+                      loading={isSubmitting}
```

Comment:
Should this be `isUpdatingDiskSize` instead of `isSubmitting`? I wasn't getting any loading state